packages/fields/src/module.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module '@wordpress/editor';
+declare module '@wordpress/patterns';
```

Comment:
What is this about?

Reply:
Why didn't we need something like it before (when the actions were in editor).

Reply:
Because we are using `@ts-ignore` to ignore the errors. Both solutions aren't great and given that we are already using `@ts-ignore`, let's remove this file!